### ISSUE TITLE
why is %20 turned into a space when using Router.navigate?

### ISSUE THREAD

**Issue by davis** [jashkenas/backbone#4059]
This issue has been here for a while, I couldn't find anything issues, so I went ahead and made a test repo. Was hoping you could provide some insight as to why %20s are being turned into spaces.

https://github.com/davis/backbone-router-test

Thanks!

**Comment by davis** [comment]
### Test case

```js
var Backbone = require('backbone');

var router = new Backbone.Router();
Backbone.history.start();

router.navigate('space%20space');
```

#### Expected result

Url looks something like `http://localhost:3000/space%20space`

#### Actual result

Url looks like `http://localhost:3000/space space`

### Why this is a problem

This breaks URLs when copy pasting them, etc, and in general is inconsistent behavior, because none of the other URLencoded chars get decoded.

### How to see it for yourself

```sh
git clone git@github.com:davis/backbone-router-test.git
cd backbone-router-test
open index.html
```

**Comment by lfilho** [comment]
Not sure but are you using jQuery 3 under the hood? It might be it: https://jquery.com/upgrade-guide/3.0/#breaking-change-jquery-param-no-longer-converts-20-to-a-plus-sign

**Comment by Flamefork** [comment]
Looks like #4025

**Comment by francescq** [comment]
Any plan on releasing a version containing #4025 fix?

Thanks!

**Comment by jgonggrijp** [comment]
Seems to have been fixed in #4026.
